Szymon/Zadanie7/src/test/java/Szymon7/User.java

```diff
@@ -0,0 +1,67 @@
+package Szymon7;
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+public class User {
+
+    private String username;
```

Comment:
Nie ma tu nic zdroznego na razie - mamy defaultowe wartosci w klasie.
Ale pamietaj, zeby nie poszlo to nigdy w strone, gdzie tworzysz rozne klasy tylko po to, zeby miec rozne dane.

W takiej klasie raczej zawsze potrzebne sa settery, lub builder, zeby moc ustawiac rozne dane.
Ta klasa powinna byc takim 'naczyniem' na dane, ktore moga byc rozne w zaleznosci od potrzeb.

Reply:
ok, ciekawa uwaga z 'naczyniem'